.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Subject of the issue
+Describe your issue here.
+
+### Your environment
```

Comment:
I would add an extra point here to list all the installed extensions and its versions since we are deploying really frequently.